flocker/control/httpapi.py

```diff
@@ -52,6 +52,13 @@
 CONTAINER_PORT_COLLISION = make_bad_request(
     code=CONFLICT, description=u"A specified external port is already in use."
 )
+LINK_PORT_COLLISION = make_bad_request(
+    code=CONFLICT,
+    description=u"Local port links in a container must be unique."
```

Comment:
That's a little confusing... maybe "The local ports in a container's links must be unique."